sources/academy/tutorials/node_js/debugging_web_scraper.md

```diff
@@ -49,40 +49,42 @@ You can easily get all information you need by running a small snippet of your p
 ```js
 results = [];
 $('.my-list-item').each((i, el) => {
- results.push({
- title: $(el).find('.title').text().trim(),
- // other fields
- })
-})
+    results.push({
+        title: $(el).find('.title').text().trim(),
+        // other fields
+    });
+});
 ```
 
 Now the `results` variable stays on the page and you can do whatever you wish with it. Usually, simply log it to analyze if your scraping code is correct. Writing a single expression will also log it in a browser console.
 
 ```js
-results
+results;
```

Comment:
shouldn't this be `console.log(results)`?

Reply:
This section is about printing them in browser dev console. It is a bit weird with the semicolon but silences the linter.